README.md

```diff
@@ -64,38 +67,38 @@ rm blender-2.81a-linux-glibc217-x86_64.tar.bz2
 ```
 
 Add blender library path to `$LD_LIBRARY_PATH`:
-```bash
+```shell
 echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/blender/lib' >> ~/.bashrc
 ```
 
 ##### 2. Copy required python libraries to blender
 Copy python libraries to the python's blender site-packages:
-```bash
-sudo cp -r /opt/conda/envs/plant_scanner/lib/python3.7/site-packages/markupsafe/ /opt/blender/2.81/python/lib/python3.7/site-packages/
-sudo cp -r /opt/conda/envs/plant_scanner/lib/python3.7/site-packages/flask/ /opt/blender/2.81/python/lib/python3.7/site-packages/
-sudo cp -r /opt/conda/envs/plant_scanner/lib/python3.7/site-packages/itsdangerous/ /opt/blender/2.81/python/lib/python3.7/site-packages/
-sudo cp -r /opt/conda/envs/plant_scanner/lib/python3.7/site-packages/click/ /opt/blender/2.81/python/lib/python3.7/site-packages/
-sudo cp -r /opt/conda/envs/plant_scanner/lib/python3.7/site-packages/werkzeug/ /opt/blender/2.81/python/lib/python3.7/site-packages/
-sudo cp -r /opt/conda/envs/plant_scanner/lib/python3.7/site-packages/jinja2/ /opt/blender/2.81/python/lib/python3.7/site-packages/
-sudo cp -r /opt/conda/envs/plant_scanner/lib/python3.7/site-packages/imageio/ /opt/blender/2.81/python/lib/python3.7/site-packages/
+```shell
+sudo cp -r /opt/conda/envs/plant_imager/lib/python3.7/site-packages/markupsafe/ /opt/blender/2.81/python/lib/python3.7/site-packages/
+sudo cp -r /opt/conda/envs/plant_imager/lib/python3.7/site-packages/flask/ /opt/blender/2.81/python/lib/python3.7/site-packages/
+sudo cp -r /opt/conda/envs/plant_imager/lib/python3.7/site-packages/itsdangerous/ /opt/blender/2.81/python/lib/python3.7/site-packages/
+sudo cp -r /opt/conda/envs/plant_imager/lib/python3.7/site-packages/click/ /opt/blender/2.81/python/lib/python3.7/site-packages/
+sudo cp -r /opt/conda/envs/plant_imager/lib/python3.7/site-packages/werkzeug/ /opt/blender/2.81/python/lib/python3.7/site-packages/
+sudo cp -r /opt/conda/envs/plant_imager/lib/python3.7/site-packages/jinja2/ /opt/blender/2.81/python/lib/python3.7/site-packages/
+sudo cp -r /opt/conda/envs/plant_imager/lib/python3.7/site-packages/imageio/ /opt/blender/2.81/python/lib/python3.7/site-packages/
 ```
 
 ##### 3. Install ROMI sources
-We will now finish by cloning and installing the ROMI sources for `romiscanner`, `romiscan` and `romidata`:
-```bash
-ROMISCAN_BRANCH=dev
-ROMIDATA_BRANCH=dev
-ROMISCANNER_BRANCH=master
+We will now finish by cloning and installing the ROMI sources for `plant-imager`, `plant-3d-vision` and `plantdb`:
+```shell
+PLANT3DVISION_BRANCH=dev
```

Comment:
Please notice that romitask hase replaced plant-3d-vision as a dependency.

Reply:
Done